scss/_orange-navbar.scss

```diff
@@ -0,0 +1,62 @@
+// Contents
+//
+// Supra bar
+
+
+// Supra bar
+//
+// Brand exclusive thin navbar that can be used with orange navbar
+.supra {
+  display: none;
+  padding-top: 0;
+  padding-bottom: 0;
+  border-bottom: $navbar-orange-border-bottom solid $gray-700;
+
+  @include media-breakpoint-up(md) {
+    display: flex;
+    .container {
+      padding-right: $navbar-orange-padding-x;
+      padding-left: $navbar-orange-padding-x;
+    }
+  }
+
+  .navbar-nav .nav-link {
+    padding: #{map-get($spacers, 2) + $navbar-supra-padding-y} $navbar-supra-padding-x $navbar-orange-link-padding-bottom;
```

Comment:
variable need to mean something. Here maybe navbar->supra->nav-link->padding-*
Like for https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/2028fd76720bebe61e8dd9e778117bc2a23a2531/scss/_variables.scss#L1237
And remove sass calc with spacer from this file, create a dedicated variable if needed